docs/mp/guides/05_security-oidc.adoc

```diff
@@ -93,7 +90,7 @@ bin/standalone.bat
 
 Keycloak runs on localhost:8080 by default.
 
-===== Create an admin user
+=== Create an admin user
```

Comment:
Minor point: Is the convention to use normal capitalization for _all_ headers? For headers above a certain level? There are several in this file which remain lower case, whereas in tracing (for example) lower-case headers become capitalized. Those (in tracing) I'm referring to were also promoted, hence my question about whether the convention depends on the header level.